compute-budget/src/compute_budget.rs

```diff
@@ -133,98 +120,143 @@ pub struct ComputeBudget {
 
 impl Default for ComputeBudget {
```

Comment:
where do we still use `ComputeBudget`? do we need the transformations between this and new types, or would it be better to just convert the remaining usages of `ComputeBudget` to the new types?

Reply:
It's currently optionally stored in Bank. And can be read from snapshot. We can remove it once we determine that/if it's always `None`.

Reply:
it doesn't seem it can be read from a snapshot: https://github.com/anza-xyz/agave/blob/master/runtime/src/bank.rs#L1792

If loading from a snapshot it is set via the `runtime_config` arg to the various `Bank::new` variants.